packages/ui_components_lib/lib/components/input/currency_text_input_formatter.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/services.dart';
+import 'package:money2/money2.dart';
+
+import 'package:ui_components_lib/components/input/input.dart';
+
+class CurrencyTextInputFormatter extends TextInputFormatter {
```

Comment:
Can you add some getter like `Fixed? currentValue`? So we will be able to get direct `Fixed` value without parsing it later

Reply:
No, we can't because we should not use formatter or validator as a controller because they can be shared between multiple input sources and shouldn't save source-related state inside themselves.

Reply:
Tamam